coalaip_bigchaindb/utils.py

```diff
@@ -16,8 +16,81 @@ def reraises_if_not(*args, **kwargs):
                 return func(*args, **kwargs)
             except Exception as ex:
                 if not isinstance(ex, allowed_exceptions):
-                    raise PersistenceError(error=ex)
+                    raise PersistenceError(error=ex) from ex
                 else:
                     raise
         return reraises_if_not
     return decorator
+
+
+def order_transactions(transactions):
```

Comment:
cc @libscott in case you were curious about the result of our conversation earlier about reconstructing a list of transaction's orders.